src/app/contextBroker/contextBroker.cpp

```diff
@@ -1771,14 +1763,14 @@ int main(int argC, char* argV[])
     LM_T(LmtHttps, ("httpsKeyFile:  '%s'", httpsKeyFile));
     LM_T(LmtHttps, ("httpsCertFile: '%s'", httpsCertFile));
 
-    restInit(rsP, ipVersion, bindAddress, port, mtenant, connectionMemory, maxConnections, reqPoolSize, rushHost, rushPort, allowedOrigin, httpsPrivateServerKey, httpsCertificate);
```

Comment:
It was suggested ,on the issue https://github.com/telefonicaid/fiware-orion/issues/1575, to change from  numerical values to definitions in common/limits.h, in restInit function.